.circleci/config2.yml

```diff
@@ -0,0 +1,945 @@
+version: 2
```

Comment:
:scissors:

Reply:
oops. I always do that :man_facepalming: